lib/logic/level.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:hive/hive.dart';
+
+//this class is used to manage the Hive storage box in relation to the statistics page.
+//this class only has class methods.
+class Stats {
```

Comment:
ueberlege dir, ob du nicht die Box box generell zu einer Klassenvariable machen moechtest, damit du nicht immer box zusaetzlich uebergeben musst. Dann kannst du stattdessen einfach einmal am Anfang einen Konstruktor mit der box aufrufen und dann in jeder Funktion die Klassenbox nutzen.

Reply:
Koennte mir auch vorstellen, dass eine Erweiterung von der Box Klasse cool waere:
class Statsbox extends Box{
}
Dann koenntest du immer mit this auf die Box zugreifen.